src/test/java/com/github/paweladamski/httpclientmock/HttpClientVerifyTest.java

```diff
@@ -226,4 +233,103 @@ public void should_check_header() throws IOException {
     httpClientMock.verify().get("/login").withHeader("User-Agent", "IE").notCalled();
   }
 
+  @Test
+  public void withFormParameter() throws IOException {
```

Comment:
Please make this test more compact, I think you can test the same thing using a smaller number of test requests.

Reply:
I removed some of the code (see 06bbc4b5592745b41bead8038787dc70d90d1629). Is that better?

I suppose most of it is redundant, since the bulk of the functionality is in the `UrlEncodedFormCondition` class.